arviz/rcparams.py

```diff
@@ -93,23 +102,37 @@ def _validate_boolean(value):
     return value is True or value == "true"
 
 
-def _validate_tuple(value):
-    """Validate value is a tuple of size 2."""
-    if value is None:
-        return value
+def _validate_float_iterable(value, length):
+    """Validate value is a float iterable of given length."""
+    if value is None or isinstance(value, str) and value.lower() == "none":
+        return None
     elif value == "auto":
         return value
-    elif isinstance(value, tuple) and len(value) == 2:
-        return value
+    elif isinstance(value, str):
+        try:
+            val = [_validate_float_or_none(v.strip("( )")) for v in value.split(',') if v.strip()]
+            if len(val) == length:
+                return tuple(val)
+            else:
+                raise ValueError("Iterable must be of length: {}".format(length))
+        except Exception:
+            raise
+    elif np.iterable(value) and not isinstance(value, (set, frozenset)):
+        val = [_validate_float_or_none(v) for v in value]
+        if len(val) == length:
+            return tuple(val)
+        else:
+            raise ValueError("Iterable must be of length: {}".format(length))
     else:
-        raise ValueError("Only tuple of size 2 is valid.")
+        raise ValueError("Only float iterable values are valid")
```

Comment:
It would be great to include `length` in the error message. a `.format` should be enough

Reply:
Alright